services/src/atdd/src/main/java/org/openkilda/topo/TopoSlug.java

```diff
@@ -31,51 +27,91 @@
     public static final String NULL_ENDPOINT = NULL_PORT + DELIM + "Q.0";
     public static final String NULL_LINK = "L" + DELIM + NULL_ENDPOINT + EP_DELIM + NULL_ENDPOINT;
 
-    public static final String toString(Switch aSwitch) {
-        return "S." + aSwitch.getId();
+    /**
+     * Returns a string representation of the switch.
+     *
+     * @param theSwitch the switch.
+     * @return string representation.
+     */
+    public static final String toString(Switch theSwitch) {
+        return "S." + theSwitch.getId();
```

Comment:
A-Switch is our internal term. We do not have "the-switch" term.

Reply:
Well, style checker ask to change `aSwitch` name. We need to think how to change style rules then.

Reply:
The exception in stylecheck config was added.